app/src/main/kotlin/io/element/android/x/info/Logs.kt

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright (c) 2023 New Vector Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.element.android.x.info
+
+import io.element.android.x.BuildConfig
+import timber.log.Timber
+import java.text.SimpleDateFormat
+import java.util.*
+
+class Logs {
```

Comment:
I wouldn't create a class for that just the method

Reply:
OK, I will change. In the future, we may want to inject classes in the constructor, to get the SDK version or instance.